Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -1777,6 +2124,594 @@ RefPtr<StyleValue> Parser::parse_box_shadow_value(ParsingContext const& context,
     return BoxShadowStyleValue::create(offset_x, offset_y, blur_radius, color);
 }
 
+RefPtr<StyleValue> Parser::parse_flex_value(ParsingContext const& context, Vector<StyleComponentValueRule> const& component_values)
+{
+    auto is_flex_grow_or_shrink = [](StyleValue const& value) -> bool {
+        if (value.is_numeric())
+            return true;
+        return false;
+    };
+
+    auto is_flex_basis = [](StyleValue const& value) -> bool {
+        if (value.is_length())
```

Comment:
`auto` is covered by this?

Reply:
It is currently, but that's not robust. I'll change it.